src/main/java/org/folio/dataexp/controllers/DownloadRecordController.java

```diff
@@ -52,7 +53,7 @@ public ResponseEntity<Resource> downloadRecordById(
         idType,
         suppress999ff
     );
-    var fileName = recordId + formatPostfix + ".mrc";
+    var fileName = "%s%s.%s".formatted(recordId, formatPostfix, Constants.MARC_FILE_SUFFIX);
```

Comment:
"%s%s.%s" pattern can be stored in Constants as named constant because it is used in couple of places in code.

Reply:
I moved `%s-%s.%s` into a constants class; after a minor change in this controller, it turns out `%s%s.%s` is only used in one place elsewhere.